src/transformers/configuration_dpr.py

```diff
@@ -36,12 +36,73 @@ class DPRConfig(BertConfig):
     It is used to instantiate the components of the DPR model.
 
     Args:
+        vocab_size (:obj:`int`, optional, defaults to 30522):
```

Comment:
Not related to this PR per-se but since this docstrings is added, let's add it right :-)
Could we replace all optional by `optional` in the below?